### PR TITLE
202101 public release

### PR DESCRIPTION
**Pull request by laurenchilutti**
This pull request will publish the code changes developed by the FV3 Team at GFDL for the **202101 public release**.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,20 @@ SHiELD_driver contains the SHiELD Physics component of the SHiELD atmosphere mod
 
 More information is available on the [GFDL SHiELD page](https://www.gfdl.noaa.gov/shield/).
 
+# History & Attribution
+
+```FV3GFS```, ```GFS_layer```, and ```gsmphys``` contain derivative code (baseline 2017 GFS) plus innovations developed by GFDL during and subsequent to the NGGPS Phase II and III (competition and implementation phases).
+
+```IPD_layer``` was developed by GFDL personnel, with community input, under funding from NWS Office of Science and Technology Integration (mostly superseded within the UFS by the CCPP [framework](https://github.com/NCAR/ccpp-framework) and [physics)](https://github.com/NCAR/ccpp-physics).
```
Comment (lharris4) [inline README.md:11]: The parenthetical statement about CCPP isn't necessary and gets too much into the "dirty laundry".

**Review by lharris4 — CHANGES_REQUESTED** [review]
Please see my proposed revision for README.md. Otherwise the code looks fine since it is identical to the code I had earlier approved.

File: README.md (hunk 1 above)
Comment (laurenchilutti) [reply README.md]: This is updated in the latest commit

**Review by lharris4 — APPROVED** [review]

**Comment by lharris4** [comment]
Thanks. I have approved the pull request.

**Review by bensonr — APPROVED** [review]

**Pull request merged**